ros/finalarm_description/urdf/finalarm_description.urdf.xacro

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="finalarm_description">
+  <xacro:arg name="use_gazebo" default="false" />
+
+  <xacro:if value="$(arg use_gazebo)">
```

Comment:
Set this up so this file contains the source of truth for the robot. Added a few nice macros to the end here. Not sure if it's as easy as swapping out the STL files for the actual description, but it's worth considering parameterizing this further so we can reuse it for the simplified robot.